app/scripts/views/mixins/avatar-mixin.js

```diff
@@ -32,10 +35,13 @@ define(function (require, exports, module) {
       // doesn't have a profile image if it's not cached
       if (self._shouldShowDefaultProfileImage(account)) {
         self.$(wrapperClass).addClass('with-default');
+      } else if (showSpinner) {
+        self._addLoadingSpinner(self.$(wrapperClass).addClass('with-spinner'));
```

Comment:
Should the `addClass('with-spinner')` be done in the `_addLoadingSpinner` method?